e2etests/web/text_editing/text_editing.iml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We do not put IDE files in the repositories.

Reply:
done